### ISSUE TITLE
Timm - vit_base_patch16_224

### ISSUE THREAD

**Issue by StrongTanisha** [StrongResearch/isc-demos#48]
**Source / repo**

https://github.com/huggingface/pytorch-image-models

**Model description**

[DESCRIPTION]

**Dataset**

imagenet
[URL]

**Literature benchmark source**

[URL]

**Literature benchmark performance**

[DESCRIPTION]
[VALUE/S]

**Strong Compute result achieved**

[VALUE/S]

**Basic training config (as applicable)**

Nodes: 12
Epochs: 350
Effective batch size: [N]
Learning rate: [L]
Optimizer: [OPT]

**Logs gist**

[URL]
